### PR TITLE
unit tests for get_quest_participants

### LINKED ISSUE #604: test: `/get_quest_participants?quest_id=${id}`
read description here - https://github.com/starknet-id/starknet.quest/issues/602#issue-2255907921

### PR DESCRIPTION
**Pull request by muheebyusufbaba1**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Testing

Resolves: #604

### CODE REVIEW

**Review by ayushtom — COMMENTED** [review]
can you also add response format in the return type of the api call so we can consume those typings on the front

**Comment by muheebyusufbaba1** [comment]
@ayushtom

**Comment by ayushtom** [comment]
@muheebyusufbaba1 you have build issues
you can check the issue locally by running npm run build and seeing what is the issue and fix it
once done i think your PR is good to go for merge

**Comment by ayushtom** [comment]
@muheebyusufbaba1  i will be assigning someone else to finish this PR today. If you can then please commit the changes here so i can review and merge or else we give it to someone who can close the issue

**Comment by muheebyusufbaba1** [comment]
I'll do this right away, working on it

File: app/quest-boost/[boostId]/page.tsx [hunk 1]
```diff
@@ -45,8 +45,8 @@ export default function Page({ params }: BoostQuestPageProps) {
       let total = 0;
       await Promise.all(
         questIds?.map(async (questID) => {
-          const res = await getQuestParticipants(questID);
-          if (res?.count) total += res?.count;
+          const res = (await getQuestParticipants(questID)) as QuestParticipantsDocument;
```
Comment (ayushtom) [inline app/quest-boost/[boostId]/page.tsx:48]: is this type hardcoding really required

**Review by ayushtom — CHANGES_REQUESTED** [review]

File: app/quest-boost/[boostId]/page.tsx (hunk 1 above)
Comment (muheebyusufbaba1) [reply app/quest-boost/[boostId]/page.tsx:48]: Build is failing when I take it out

Comment (ayushtom) [reply app/quest-boost/[boostId]/page.tsx:48]: got it

**Review by ayushtom — APPROVED** [review]
lgtm

**Pull request merged**